DEVELOPMENT.md

```diff
@@ -519,62 +532,96 @@ yarn dist
 
 Where:
   - `CSC_NAME` is the name of the signing certificate
-  - `APPLE_ID` is the developer's Apple developer ID
-  - `APPLE_ID_PASSWORD` is an app specific token generated from your Apple ID
-    in the developer portal
+  - `APPLE_ID` is your Apple developer ID
+  - `APPLE_ID_PASSWORD` is the app specific token generated from your Apple ID
 
 **Note**: building a release might take a while, especially the notarization
 step, because it has to upload the final package to the Apple notarization
 server. Make sure you're on a stable internet connection.
 
-Once the package is built, signed and notarized you can upload it to the web
-using [`gsutil`][gs]:
+Once the package is notarized you can upload it to the web using
+[`gsutil`][gs]:
 
 ```bash
-gsutil cp dist/radicle-upstream-0.1.2.dmg gs://releases.radicle.xyz
+gsutil cp dist/radicle-upstream-X.X.X.dmg gs://releases.radicle.xyz
 ```
 
 To be able to upload packages to the GCS bucket you will need the appropriate
 permissions. Reach out to a co-worker if you don’t have them.
 
 After this you also need to obtain and upload the Linux package.
 
+FIXME(rudolfs): this doesn't work, we don't build images off of tags.
+
 ```bash
-curl -sSLO https://builds.radicle.xyz/radicle-upstream/v0.1.2/radicle-upstream-0.1.2.AppImage
-gsutil cp radicle-upstream-0.1.2.AppImage gs://releases.radicle.xyz
+curl -sSLO https://builds.radicle.xyz/radicle-upstream/vX.X.X/radicle-upstream-X.X.X.AppImage
+gsutil cp radicle-upstream-X.X.X.AppImage gs://releases.radicle.xyz
 ```
 
 After all the packages are uploaded, update the links to those binaries on the
 [radicle.xyz download][rd] and [docs.radicle.xyz/docs/getting-started][gs]
 pages and rebuild/deploy the websites.
 
-The final step is to announce the new release on our public channels:
-  - https://twitter.com/radicle
-  - https://radicle.community/c/announcements
-
-Here's a template for the annoucement:
+The final step is to announce the new release on our public channels (**make
+sure to update all the versions and links**):
 
-```
-Radicle Upstream v0.0.11 is out! 🎉
-
-All the changes that went into this release, you'll find here:
-  https://github.com/radicle-dev/radicle-upstream/blob/master/CHANGELOG.md#0011-2020-05-25
-
-And here are packages for all our supported platforms:
-  - macOS:
-    https://releases.radicle.xyz/radicle-upstream-0.1.3.dmg
-  - Linux:
-    https://releases.radicle.xyz/radicle-upstream-0.1.3.AppImage
-
-For support you can reach us here:
-  https://radicle.community/c/help
-  https://matrix.radicle.community/#/room/#support:radicle.community
-
-If you encounter a bug, please open an issue here:
-  https://github.com/radicle-dev/radicle-upstream/issues
-```
-
-[an]: #setting-up-apple-notarization
+  - https://radicle.community/c/announcements
+    - subject:
+        Radicle Upstream vX.X.X is out! 🎉`
```

Comment:
```suggestion
        Radicle Upstream vX.X.X is out! 🎉
```

Reply:
Good catch!

The formatting was also busted, fixed in:
https://github.com/radicle-dev/radicle-upstream/pull/1485/commits/c0c438543ebd746977f3f3cb7ba67a70dc75d7de
https://github.com/radicle-dev/radicle-upstream/pull/1485/commits/48229e541009d56b5c3eed2525b8ab4d40a06cd0

<img width="1056" alt="Screenshot 2020-12-16 at 14 56 38" src="https://user-images.githubusercontent.com/158411/102357889-1fc89d80-3faf-11eb-99dc-24642d7be66f.png">